thread_options.h

```diff
@@ -203,7 +203,7 @@ struct thread_options {
 	enum fio_memtype mem_type;
 	unsigned int mem_align;
 
-	unsigned long long max_latency;
+	unsigned long long max_latency[DDIR_RWDIR_CNT];
```

Comment:
We're OK with changing this data structure and not ramping FIO_SERVER_VER in server.h because the last ramp in https://github.com/axboe/fio/commit/a87c90fd72823d5438d724e5a57ced8d1f7bed3f has yet to make it into an official release.

Reply:
Thanks a lot for the quick review! I will shorten the title and add the details in commit message as suggested. I will also add my name and email address in Signed-off-by.